app/src/main/java/com/kickstarter/ui/viewholders/CommentViewHolder.kt

```diff
@@ -0,0 +1,63 @@
+package com.kickstarter.ui.viewholders
+
+import android.graphics.Typeface
+import android.util.Pair
+import android.view.View
+import com.kickstarter.R
+import com.kickstarter.databinding.CommentCardViewBinding
+import com.kickstarter.libs.transformations.CircleTransformation
+import com.kickstarter.libs.utils.CommentUtils
+import com.kickstarter.libs.utils.DateTimeUtils
+import com.kickstarter.libs.utils.ObjectUtils
+import com.kickstarter.models.Comment
+import com.kickstarter.models.Project
+import com.squareup.picasso.Picasso
+
+class CommentViewHolder(private val binding: CommentCardViewBinding) : KSViewHolder(binding.root) {
+    private var comment: Comment? = null
+    private val ksString = environment().ksString()
+    private var project: Project? = null
+
+    @Throws(Exception::class)
+    override fun bindData(data: Any?) {
+        val projectAndComment = ObjectUtils.requireNonNull(data as? Pair<*, *>?)
+        project = ObjectUtils.requireNonNull(projectAndComment.first as? Project, Project::class.java)
+        comment = ObjectUtils.requireNonNull(projectAndComment.second as? Comment, Comment::class.java)
+    }
+
+    override fun onBind() {
+        val context = context()
+        val currentUser = environment().currentUser().observable()
+
+        binding.creatorLabel.visibility = View.GONE
+        binding.userLabel.visibility = View.GONE
+        comment?.let { comment ->
+            currentUser
+                .compose(bindToLifecycle())
+                .subscribe { user ->
+                    if (CommentUtils.isUserAuthor(comment, project?.creator())) {
+                        binding.creatorLabel.visibility = View.VISIBLE
+                    } else if (CommentUtils.isUserAuthor(comment, user)) {
+                        binding.userLabel.visibility = View.VISIBLE
+                    }
+                }
+
+            Picasso.with(context).load(comment.author()?.avatar()?.small())
```

Comment:
Since `small()` is `@Nullable`, we should be checking that this url is not null or else Picasso will cause the app to crash.